dask_sql.Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+
+FROM gpuci/miniconda-cuda:$CUDA_VER-devel-$LINUX_VER
+
+ARG CUDA_VER=11.2
+ARG PYTHON_VER=3.8
+ARG NUMPY_VER=1.21
+ARG RAPIDS_VER=21.08
+ARG UCX_PY_VER=0.21
+ARG JAVA_VER=11
+
+ADD https://raw.githubusercontent.com/dask-contrib/dask-sql/main/conda.txt /dask_sql_requirements.txt
+
+RUN conda config --set ssl_verify false
+
+RUN conda install -c gpuci gpuci-tools
+
+RUN gpuci_conda_retry install -c conda-forge mamba
+
+RUN gpuci_mamba_retry env create -n dask_sql --file /dask_sql_requirements.txt -c conda-forge
+
+RUN gpuci_mamba_retry install -y -n dask_sql -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
+    cudatoolkit=$CUDA_VER \
+    cudf=$RAPIDS_VER \
+    dask-cudf=$RAPIDS_VER \
+    numpy=$NUMPY_VER \
+    openjdk=$JAVA_VER \
+    # following requirements are for postgres testing; might not need them
+    sqlalchemy>=1.4.23 \
+    pyhive>=0.6.4 \
+    psycopg2>=2.9.1 \
+    ciso8601>=2.2.0 \
+    tpot>=0.11.7 \
+    mlflow>=1.19.0 \
+    docker-py>=5.0.0
+    
+RUN conda activate dask_sql && python -m pip install fugue[sql]>=0.5.3
+
+RUN docker pull bde2020/hive:2.3.2-postgresql-metastore
```

Comment:
Not too familiar with making Dockerfiles; is this the preferred way to pull another Docker image from within a Dockerfile?

Reply:
No, usually you have on `FROM` statement at the top which sets the base image. What are you trying to do with this line?

Reply:
It looks like some of the PostgreSQL testing dependencies are pulled from docker images:

https://github.com/dask-contrib/dask-sql/blob/4d5f7dd4c133f7f2c584db2492c8f0ef3cba7e31/.github/workflows/test.yml#L104-L105

However, we shouldn't need these lines if we pass on adding PostgreSQL dependencies for now